InternshipBE/DAL/Repositories/LocationRepository.cs

```diff
@@ -0,0 +1,25 @@
+﻿using DAL.Database;
+using DAL.Entities;
+using DAL.Interfaces;
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace DAL.Repositories
+{
+    public class LocationRepository : GenericRepository<Country>, ILocationRepository
+    {
+        public LocationRepository(ApplicationDbContext context): base(context)
+        {            
+        }
+
+        public async Task<List<Country>> GetLocationsAsync()
+        {
+            var locations = await _context.Countries.ToListAsync();
+            
+            
+
+            return null;
```

Comment:
why we return null?

Reply:
This is so that Eugene can switch to this thread. We'll fix it all)